src/main/java/io/jenkins/plugins/conventionalcommits/ConventionalCommits.java

```diff
@@ -27,13 +27,18 @@ public Version nextVersion(Version in, List<String> commits) {
     }
 
     private boolean breakingChangeFooter(String commit){
-        int startIndex = commit.lastIndexOf("\n");
 
-        String result = commit;
+        boolean result = false;
+        String[] lines = commit.split("[\\r\\n]+");
 
-        if(startIndex!=-1 && startIndex!= commit.length()){
-            result = commit.substring(startIndex+1);
+        for(String line: lines){
+            if (line.startsWith("BREAKING CHANGE:") || line.startsWith("BREAKING-CHANGE:")) {
```

Comment:
What would happen if the line was `breaking change`  or `Breaking Change`?

Reply:
According to the conventional commits guidelines only `BREAKING CHANGE` and `BREAKING-CHANGE` (case sensitive) are considered as keywords for breaking change.  Others are just text and can be ignored. 
Please correct me if I am wrong

Reply:
That's my understanding as well, would it make sense to trigger a notice in such case like '' looks like you specified BreakIng ChAnGed'' which is not compliant

Reply:
Great, I did want to make sure that there wasn't anything that would trip up the parsing.